core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsBillingProjectComponent.scala

```diff
@@ -99,6 +99,13 @@ final case class BillingAccountChange(id: Long,
                                       outcome: Option[Outcome]
 )
 
+final case class WorkspaceManagerResourceMonitor(jobControlId: UUID,
```

Comment:
I would find it clearer if this was named `WorkspaceManagerResourceMonitorRecord`.